### ISSUE TITLE
[Feature Request] Add Standard Deviation to summarystats()?

### ISSUE THREAD

**Issue by noamsgl** [JuliaStats/StatsBase.jl#693]
Feature Request: Add STD to summarystats()...

https://github.com/JuliaStats/StatsBase.jl/blob/a0e6f1e807a84a09b5f74431bb0099f4aaed5ae0/src/scalarstats.jl#L555

**Comment by nalimilan** [comment]
I agree that it's surprising not to print the standard deviation here. Do you feel like making a pull request to add it?

**Comment by itsdebartha** [comment]
@nalimilan is this a dormant issue or still open for contribution?

I was thinking how about a `summarystat()` just like the `summary` function in R. It'll return different values for string/factor types just like R does...

**Comment by nalimilan** [comment]
We need more opinions to decide what's best. In DataFrames (https://github.com/JuliaData/DataFrames.jl/pull/2459), we decided not to report standard deviations and quartiles by default so that the output fits in the screen width: one needs to do `describe(df, :detailed)` to get them. Here screen width isn't a problem and we already report quartiles, so maybe we could print the standard deviation too.

@bkamins @pdeffebach What do you think?

**Comment by bkamins** [comment]
In general I almost always want to see `std`, so I would like to have this change. The only issue is that it would be breaking. I am not sure what decision would be best. Maybe we can consider it to be mildly breaking and go for it?

**Comment by pdeffebach** [comment]
I agree I almost always want `std`. I would maybe call it mildly breaking? Its really only useful in interactive work.

**Comment by nalimilan** [comment]
That would only change the printing, so that's considered non-breaking I think?

**Comment by nalimilan** [comment]
Do we consider that adding a new field to an object is breaking though? That sounds quite restrictive.

**Comment by bkamins** [comment]
OK - let us add it.

**Comment by itsdebartha** [comment]
Created #858
